src/prost/google.protobuf.rs

```diff
@@ -109,6 +114,90 @@ pub struct ExtensionRangeOptions {
     /// The parser stores options it doesn't recognize here. See above.
     #[prost(message, repeated, tag = "999")]
     pub uninterpreted_option: ::prost::alloc::vec::Vec<UninterpretedOption>,
+    /// For external users: DO NOT USE. We are in the process of open sourcing
```

Comment:
Interesting that it checks out different and seemingly newer google protobuf. Ideally though, we should converge on prost-types or some other basic crate to consistently represent these messages.

Reply:
> Interesting that it checks out different and seemingly newer google protobuf
Yeah I never quite understood why those change so much.

> Ideally though, we should converge on prost-types or some other basic crate to consistently represent these messages
Agreed!